editor/src/clj/editor/atlas.clj

```diff
@@ -699,6 +712,7 @@
   (property max-page-size types/Vec2
             (dynamic edit-type (g/constantly {:type types/Vec2 :labels ["W" "H"]}))
             (dynamic error (g/fnk [_node-id max-page-size] (validate-max-page-size _node-id max-page-size))))
+  (property rename-patterns g/Str)
```

Comment:
I didn't want to go down the route of creating a dynamic array right now.
It would be great to have it at some point though.